### PR TITLE
Added service that runs cron.php every 15 minutes

### PR DESCRIPTION
**Pull request by thomascobb**
Not sure if I'm doing something wrong, but I had to restart the computer before the service would start. I tried to recreate it, but it always seemed to start correctly.

I checked dmesg, and I seem to have a lot of messages like this:

[  845.275631] audit: type=1400 audit(1473627189.253:332): apparmor="DENIED" operation="exec" profile="snap.nextcloud.mysql" name="/bin/run-parts" pid=6828 comm="mysql.server" requested_mask="x" denied_mask="x" fsuid=0 ouid=0
[  845.500563] audit: type=1400 audit(1473627189.481:333): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=6982 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  845.507404] audit: type=1400 audit(1473627189.485:334): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=6982 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  845.507491] audit: type=1400 audit(1473627189.485:335): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=6982 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  845.507569] audit: type=1400 audit(1473627189.485:336): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=6982 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  845.508631] audit: type=1400 audit(1473627189.489:337): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=6982 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  845.508710] audit: type=1400 audit(1473627189.489:338): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=6982 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  845.508934] audit: type=1400 audit(1473627189.489:339): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=6982 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  845.509009] audit: type=1400 audit(1473627189.489:340): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=6982 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  845.509082] audit: type=1400 audit(1473627189.489:341): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=6982 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  867.643864] audit_printk_skb: 132 callbacks suppressed
[  867.643871] audit: type=1400 audit(1473627211.619:386): apparmor="DENIED" operation="exec" profile="snap.nextcloud.mysql" name="/bin/run-parts" pid=7181 comm="mysql.server" requested_mask="x" denied_mask="x" fsuid=0 ouid=0
[  867.651754] audit: type=1400 audit(1473627211.627:387): apparmor="DENIED" operation="exec" profile="snap.nextcloud.mysql" name="/bin/run-parts" pid=7187 comm="mysql.server" requested_mask="x" denied_mask="x" fsuid=0 ouid=0
[  870.676975] audit: type=1400 audit(1473627214.655:388): apparmor="DENIED" operation="exec" profile="snap.nextcloud.mysql" name="/bin/run-parts" pid=7215 comm="mysql.server" requested_mask="x" denied_mask="x" fsuid=0 ouid=0
[  870.785995] audit: type=1400 audit(1473627214.763:389): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=7344 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  870.792897] audit: type=1400 audit(1473627214.771:390): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=7344 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  870.792990] audit: type=1400 audit(1473627214.771:391): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=7344 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  870.793079] audit: type=1400 audit(1473627214.771:392): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=7344 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  870.793940] audit: type=1400 audit(1473627214.771:393): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=7344 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  870.794030] audit: type=1400 audit(1473627214.771:394): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=7344 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  870.794285] audit: type=1400 audit(1473627214.771:395): apparmor="DENIED" operation="ptrace" profile="snap.nextcloud.mysql" pid=7344 comm="ps" requested_mask="trace" denied_mask="trace" peer="unconfined"
[  978.570016] audit_printk_skb: 135 callbacks suppressed
[  978.570022] audit: type=1400 audit(1473627322.548:441): apparmor="DENIED" operation="open" profile="snap.nextcloud.php-fpm" name="/usr/sbin/" pid=7704 comm="find" requested_mask="r" denied_mask="r" fsuid=0 ouid=0
[  978.571569] audit: type=1400 audit(1473627322.548:442): apparmor="DENIED" operation="open" profile="snap.nextcloud.php-fpm" name="/sbin/" pid=7704 comm="find" requested_mask="r" denied_mask="r" fsuid=0 ouid=0
[  978.571739] audit: type=1400 audit(1473627322.548:443): apparmor="DENIED" operation="open" profile="snap.nextcloud.php-fpm" name="/bin/" pid=7704 comm="find" requested_mask="r" denied_mask="r" fsuid=0 ouid=0
[  978.571870] audit: type=1400 audit(1473627322.548:444): apparmor="DENIED" operation="open" profile="snap.nextcloud.php-fpm" name="/usr/games/" pid=7704 comm="find" requested_mask="r" denied_mask="r" fsuid=0 ouid=0

### CODE REVIEW

**Comment by oparoz** [comment]
These log entries look unrelated, but how did you test it was working?

1. Compiled new snap
1. Sideloaded compiled snap

Correct?

**Comment by thomascobb** [comment]
cd nextcloud-snap
snapcraft
sudo snap install --force-dangerous nextcloud_9.0.53snap3_i386.snap

**Comment by thomascobb** [comment]
Then checked the admin page and cron didn't seem to have run. I had to turn my laptop off then, but when I came to turn it on and looked at the admin page later, cron seemed to have run

**Comment by oparoz** [comment]
Thanks for confirming :)

**Comment by kyrofa** [comment]
This looks pretty good @thomascobb! It also need to be re-proposed to develop, though, so I'm going to go ahead and close it while I give it a test.

File: snapcraft.yaml [hunk 1]
```diff
@@ -66,6 +66,11 @@ apps:
     restart-condition: always
     plugs: [network, network-bind]
 
+  cron-lite:
```
Comment (kyrofa) [inline snapcraft.yaml]: Remember the snap contains everything needed by Nextcloud (mysql, apache, etc.). With that in mind, "cron-lite" is not the most descriptive of names. How about "nextcloud-cron"?

**Pull request closed without merging**